### PR TITLE
Adding platform information

### PR DESCRIPTION
**Pull request by NandoCT1014**
Adding platform information to query string using Unity's Application.platform() enum values and GDK platform directives.

### CODE REVIEW

File: targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs [hunk 1]
```diff
@@ -29,7 +29,29 @@ namespace PlayFab
 
     public static class PlayFabSettings
     {
-        static PlayFabSettings() { }
+        static PlayFabSettings() 
+        {
+#if UNITY_GAMECORE || UNITY_GAMECORE_XBOXONE || UNITY_GAMECORE_SCARLETT || MICROSOFT_GAME_CORE
+            PlatformString = "GDK";
+#else
+        switch (Application.platform)
```
Comment (tculotta) [inline targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs:37]: What about other platforms?

File: targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs [hunk 2]
```diff
@@ -29,7 +29,29 @@ namespace PlayFab
 
     public static class PlayFabSettings
     {
-        static PlayFabSettings() { }
+        static PlayFabSettings() 
+        {
+#if UNITY_GAMECORE || UNITY_GAMECORE_XBOXONE || UNITY_GAMECORE_SCARLETT || MICROSOFT_GAME_CORE
```
Comment (tculotta) [inline targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs:34]: Do we care about differentiating between these at all?

**Review by jasonsandlin — APPROVED** [review]

File: targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs (hunk 1 above)
Comment (NandoCT1014) [reply targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs:37]: Other platforms like PS5, PS4, Switch, Android, Windows, and iOS should set by converting the Platform.Runtime enum to a string directly if none of the GDK platform macros are set.

File: targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs (hunk 2 above)
Comment (NandoCT1014) [reply targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs:34]: I didn't differentiate between console and pc in this case as I'm following the pattern in setting the GDK platform macro in PF C SDK

File: targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs (hunk 1 above)
Comment (tculotta) [reply targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs:37]: Oh I see now! I'd misread the default block 👍

**Review by tculotta — APPROVED** [review]
:shipit:

**Pull request merged**